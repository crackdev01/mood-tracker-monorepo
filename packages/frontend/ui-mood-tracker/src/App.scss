.App {
  text-align: center;
}

@media (prefers-reduced-motion: no-preference) {
  .App-logo {
    height: 75px;
    margin-left: 1rem;
    width: auto;
  }
}
