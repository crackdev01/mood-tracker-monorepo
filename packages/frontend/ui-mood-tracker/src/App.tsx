import React from 'react';
import { BrowserRouter } from 'react-router-dom';

import SiteHeader from './components/SiteHeader';

function App() {
  return (
    <BrowserRouter>
      <SiteHeader />
    </BrowserRouter>
  );
}

export default App;
