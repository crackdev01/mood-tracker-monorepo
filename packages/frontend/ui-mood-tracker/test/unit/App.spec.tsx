import React from 'react';
import { render } from '@testing-library/react';

import App from '../../src/App';

jest.mock('../../src/components/SiteHeader', () => () => 'SiteHeader');

describe('App.vue', () => {
  test('renders', () => {
    const { container } = render(<App />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
