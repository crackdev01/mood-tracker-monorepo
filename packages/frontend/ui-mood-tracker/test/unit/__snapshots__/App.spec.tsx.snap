// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue renders 1`] = `SiteHeader`;
